File: Assets/Scripts/LoadLives.js
#pragma strict
var lives = 1;

function OnEnable () {
	  var initialLoad = PlayerPrefs.GetInt("initialLoad");
		if(initialLoad == 0) {
			//The first time the user loads the game, give them 5 lives.
			PlayerPrefs.SetInt("lives", 5);
		}
		lives = PlayerPrefs.GetInt("lives");
		GetComponent.<UI.Text>().text = lives.ToString();
}

File: Assets/Scripts/LoadLives.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class LoadLives : MonoBehaviour
{
    public int lives;
    public virtual void OnEnable()
    {
        this.lives = PlayerPrefs.GetInt("lives");
        this.GetComponent<UnityEngine.UI.Text>().text = this.lives.ToString();
    }

}